phoenix/core/models.py

```diff
@@ -12,6 +13,8 @@
 
 USER_MODEL = settings.AUTH_USER_MODEL
 
+ETA_PARSE_RE = re.compile(r"<?(?P<value>\d+)(?P<granularity>\w{1})")
```

Comment:
This should only allow the characters `m` or `d`. In that case you can avoid making more validation in the method

Reply:
Which right now is not done, so it would fail if someone imputs a different character

Reply:
I'm not sure what characters should be allowed. I don't know what granularity we're eventually going to use.

Reply:
It's a select field so only predefined values should come. The check in the function doesn't really makes sense just added it there as I was in rush and thought of it. This also isn't finished yet.

Reply:
I've removed the validation because it's select field. I'm not checking for specific characters in regex to allow easier addition of other characters for example `d`. This would require only adding `"d": 1440"` to mapping dict.